packages/user-ui/public/theme.css

```
@@ -42,5 +42,6 @@
   --breakpoint-tablet: 768px;
   --breakpoint-mobile: 480px;
 
+  --desktop-max-width: 1300px;
```

```suggestion
  --container-desktop: 1300px;
```
으로 정의하면 `max-w-desktop` 이런식으로 쓸수 있습니다.